Building-with-IntelliJ-IDEA.md

```diff
@@ -52,48 +62,63 @@ When first viewing the project in IntelliJ IDEA you may come across this error:
 
 This is because you do not have the [Lombok Plugin](https://plugins.jetbrains.com/plugin/6317-lombok-plugin) installed.
 
-Navigate to the **Plugins** tab under the **File > Settings** menu (**IntelliJ IDEA > Preferences** for Mac). Click the **Marketplace** button and search for **Lombok** to find it. Install the plugin and restart IntelliJ IDEA.
+Navigate to the `Plugins` tab under the `File > Settings` menu (`IntelliJ IDEA > Preferences` for Mac). Click the `Marketplace` button and search for **Lombok** to find it. Install the plugin and restart IntelliJ IDEA.
 
-![installing-lombok](img/building-with-intellij/installing_lombok.png)
+![installing_lombok.png](img%2Fbuilding-with-intellij%2Finstalling_lombok.png)
 
 Success! You should no longer be getting ``Cannot resolve symbol`` or ``Cannot resolve method`` errors.
 
-## Building the project
+
+## Configuring the project
+
+We need to make sure that the correct SDK is selected for RuneLite. To do that go to `File > Project Structure…`
+
+![project_structure.png](img%2Fbuilding-with-intellij%2Fproject_structure.png)
+
+Go to `Project Structure > Project Settings > Project` and select Java 11 as the SDK. Then press Ok to close the window.
+
+![select_java.png](img%2Fbuilding-with-intellij%2Fselect_java.png)
+
+If you are having issues with missing JDK see [Troubleshooting](#troubleshooting) section.
```

Comment:
pointing to the existing section does not really do anything if it's exactly the same as this section